tests/unit/project_test.py

```diff
@@ -770,7 +770,7 @@ def test_project_platform_value(self):
 
     def test_build_container_operation_with_timeout_func_does_not_mutate_options_with_timeout(self):
         config_data = Config(
-            version=V3_7,
+            version=V3_8,
```

Comment:
`3.7` test shouldn't be replaced. `3.8` tests should be added instead

Reply:
Fair point, I'll revert that line and push ASAP. I don't currently have any tests for 3.8.

Reply:
Done.